judge/views/stats.py

```diff
@@ -34,18 +35,21 @@ def repeat_chain(iterable):
 
 
 def language_data(request, language_count=Language.objects.annotate(count=Count('submission'))):
-    languages = language_count.filter(count__gte=1000).values('key', 'name', 'short_name', 'count').order_by('-count')
-    data = []
-    for language, color, highlight in zip(languages, chart_colors, highlight_colors):
-        data.append({
-            'value': language['count'], 'label': language['name'],
-            'color': color, 'highlight': highlight,
-        })
-    data.append({
-        'value': language_count.filter(count__lt=1000).aggregate(total=Sum('count'))['total'],
-        'label': 'Other', 'color': '#FDB45C', 'highlight': '#FFC870',
-    })
-    return JsonResponse(data, safe=False)
+    threshold = getattr(settings, 'DMOJ_STATS_THRESHOLD', 10)
```

Comment:
`DMOJ_STATS_LANGUAGE_THRESHOLD`, just in case we need more thresholds for other stats in the future.